bin/hyperspy

```diff
@@ -154,41 +125,10 @@ sys.argv = ipy_argv
 # Add the subcommand first
 if args.environment != 'terminal':
     sys.argv.insert(1, args.environment)
-# By default Hyperspy uses the qt backend, but if the user tries to
-# choose a different one using --pylab= , then we grant him the control
-add_pylab = True
-if argp is True:
-    toolkit = args.toolkit
-    if args.toolkit == "None":
-        add_pylab = False
-else:
-    toolkit = options.toolkit
-    if options.toolkit == "None":
-        add_pylab = False
-if argp is True:
-    pylab_inline = args.pylab_inline
-else:
-    pylab_inline = options.pylab_inline
-hyperspy.defaults_parser.current_toolkit = toolkit
-if toolkit in ("qt4", "wx"):
-    traits.etsconfig.etsconfig.ETSConfig.toolkit = toolkit
-else:
-    traits.etsconfig.etsconfig.ETSConfig.toolkit = "null"
-
-
-sys.argv.append('--profile=hyperspy')
-for arg in sys.argv:
-    if 'pylab' in arg:
-        add_pylab = False
-if pylab_inline is True and args.environment in ("notebook",
-                                                 "qtconsole"):
-    sys.argv.append('--pylab=inline')
-elif add_pylab is True:
-    sys.argv.append('--pylab=%s' % toolkit.replace("qt4", "qt"))
-
 if ipy_version >= LooseVersion('3.0.0') and args.environment == "notebook":
```

Comment:
A user might still pass pylab to ipython arguments... So either we leave it as it is (which fixes users mistake), or let the user deal with wrong call himself. Your call :)